src/renderer/components/SettingsModal.tsx

```diff
@@ -417,6 +417,7 @@ const PythonSettings = memo(() => {
                       },
                   ]
                 : []),
+            ...(isArmMac ? [{ label: 'CoreML', value: 'CoreMLExecutionProvider' }] : []),
```

Comment:
Wouldn't it be better to call it **GPU** instead of **CoreML**? It would make the UI more consistent. Since in all other parts of the UI hardware acceleration is also referred to as **GPU*

Reply:
How about GPU (CoreML)

Reply:
I want people to know that it's actual a MacOS GPU inference by using the actual name of the thing

Reply:
> How about GPU (CoreML)

Sounds good to me.